ui/src/app/utils/index.tsx

```diff
@@ -588,3 +588,6 @@ export const cond = <T extends unknown>(
 export const usernameWithoutDomain = (username: string) => {
   return username ? username.substring(0, username.indexOf('@')) : '';
 };
+
+export const capStringWithEllipsis = (value: string, maxLength: number) =>
```

Comment:
You may have already tried this, but if this is a styling concern you can also try using `text-overflow: ellipsis`. There is some trickiness with that though, so I'm fine with this approach as long as you've looked at the option.

Reply:
hm, looks like that is already set (line 49) but it does not seem to be having any effect.